sqlglot/dialects/oracle.py

```diff
@@ -328,3 +364,22 @@ def queryoption_sql(self, expression: exp.QueryOption) -> str:
         def coalesce_sql(self, expression: exp.Coalesce) -> str:
             func_name = "NVL" if expression.args.get("is_nvl") else "COALESCE"
             return rename_func(func_name)(self, expression)
+
+        def multitable_inserts_sql(self, expression: exp.MultitableInserts) -> str:
+            kind = self.sql(expression, "kind").upper()
```

Comment:
You won't need `upper` here if you apply the `self._prev.text.upper()` suggestion